### DISCUSSION TITLE
Why pitch oscilates when roll flips？

### DISCUSSION THREAD

**Discussion by lida2003** [iNavFlight/inav#9154] [Q&A]
I have ask about it in discord, but there is no answer yet.

Does any one can light me up on this?

![图片](https://github.com/iNavFlight/inav/assets/1296171/ac126664-beca-46bf-a1d3-29adba5da623)

**Comment by DzikuVx** [comment]
There is no BB file attached, no config dump. We are not prophets here

**Reply by lida2003** [reply]
The log I have downloaded from inav configurator is NOT BB file like betaflight, it's a blackbox_log_2023-06-30_084715.TXT, which is about 46MB, too larget to upload.

So I don't know how to get BB file.  Is there any way to split files？

**Reply by lida2003** [reply]
@DzikuVx  Well, I found a msc solution, which is quite different form Betaflight. https://github.com/iNavFlight/inav/blob/master/docs/USB_Mass_Storage_(MSC)_mode.md

Is there any other way to convert TXT to bb files?

**Reply by lida2003** [reply]
I think there is a bug for onboard flash when enter cli-msc.  [msc mode does not work again #9160](https://github.com/iNavFlight/inav/issues/9160)
